### PR TITLE
service: handle 3.1.0 bundle with awsconfig/v8

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: service/awsconfig/drainer_framework.go [hunk 1]
```diff
@@ -146,13 +147,30 @@ func newDrainerResourceRouter(config DrainerFrameworkConfig) (*framework.Resourc
...
+
+			GuestUpdateEnabled: config.GuestUpdateEnabled,
+			ProjectName:        config.ProjectName,
+		}
+
+		v8ResourceSet, err = v8.NewDrainerResourceSet(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
```
Comment (xh3b4sd) [inline service/awsconfig/drainer_framework.go]: Yeah this is odd we have to do it and that it is like separated. This is only to get a different resync period for certain resources. I did not come up with a cool idea to improve this.

File: service/awsconfig/v8/version_bundle.go [hunk 2]
```diff
@@ -59,7 +49,7 @@ func VersionBundle() versionbundle.Bundle {
 		Deprecated:   false,
 		Name:         "aws-operator",
 		Time:         time.Date(2018, time.February, 27, 14, 16, 0, 0, time.UTC),
-		Version:      "3.0.0",
-		WIP:          false,
+		Version:      "3.1.0",
```
Comment (xh3b4sd) [inline service/awsconfig/v8/version_bundle.go]: I think the new WIP bundle should be a patch release by default.

Comment (kopiczko) [reply service/awsconfig/v8/version_bundle.go]: I don't have opinion. Can be changed any time. I'll change to 3.0.1

File: service/awsconfig/drainer_framework.go (hunk 1 above)
Comment (kopiczko) [reply service/awsconfig/drainer_framework.go]: I find it ok. Those are separate sets, just happen to reconcile on the same object.

**Pull request merged**